build.gradle

```diff
@@ -30,6 +30,20 @@ ext.stdlibIoVersion = project.stdlibIoVersion
 ext.stdlibStringUtilsVersion = project.stdlibStringUtilsVersion
 ext.stdlibConfigVersion = project.stdlibConfigVersion
 ext.stdlibSystemVersion = project.stdlibSystemVersion
+ext.stdlibHttpVersion = project.stdlibHttpVersion
+
+ext.stdlibMimeVersion = project.stdlibMimeVersion
+ext.stdlibLogVersion = project.stdlibLogVersion
+ext.stdlibAuthVersion = project.stdlibAuthVersion
+ext.stdlibCacheVersion = project.stdlibCacheVersion
+ext.stdlibTimeVersion = project.stdlibTimeVersion
+ext.stdlibCryptoVersion = project.stdlibCryptoVersion
+ext.stdlibFileVersion = project.stdlibFileVersion
+ext.stdlibConfigVersion = project.stdlibConfigVersion
```

Comment:
Shall we remove this? config module is already added (line 31)

Reply:
will do the change